lib/graphql/resolver.ex

```diff
@@ -796,7 +796,7 @@ defmodule AshGraphql.Graphql.Resolver do
   end
 
   defp do_refetch(query, primary_key_matches) do
-    Ash.read(Ash.Query.do_filter(query, or: primary_key_matches))
+    Ash.read(Ash.Query.do_filter(query, and: primary_key_matches))
```

Comment:
why the change to and? were refetching a list of records by primary key.

Reply:
Ok, I had to reread the docs of `Map.take` now it makes more sense.

Reply:
Yeah, it took me a bit, but I reread some code and docs. The filter is automatically applied as an `and`, and the rest is an or of all the pk matches.